core/commands/ls.go

```diff
@@ -93,43 +81,39 @@ The JSON output contains type information.
 			dserv = merkledag.NewDAGService(bserv)
 		}
 
-		paths := req.Arguments()
+		paths := req.Arguments
```

Comment:
There really isn't a need to set this anymore, we can just use it directly in the loop below.